pwa/src/components/applications/applicationsTable.tsx

```diff
@@ -96,6 +96,17 @@ export default function ApplicationsTable() {
                       headerName: "Description",
                       field: "description",
                     },
+                    {
+                      headerName: "Redocly",
+                      field: "name",
+                      renderCell: () => {
+                        return (
+                          <Link to={`https://redocly.github.io/redoc/?url=http://localhost/openapi.yaml&nocors`} target="__blank">
```

Comment:
Also, I think it would be better to store this URL somewhere else, it's not content that belongs to the application table I think. Might be better placed somewhere in (e.g.) the environment variables.